index.html

```diff
@@ -0,0 +1,144 @@
+    </div>
+    <div class="service-item-desc">
+      <h2 class="service-item-heading">
+          Premium Materials
+      </h2>
+      <p class="service-item-text">Ort trombones use the shiniest brass which is sourced locally. This will increase the logevity of you purchase.
+
+      </p>
+    </div>
+    </li>
+    <li class="services-item">
+       <div class="service-item-img">
+      <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTti2je5jP4NJyie2Gjynf4ZklgkTOOGdloLQ&usqp=CAU" alt="">
+    </div>
+    <div class="service-item-desc">
+      <h2 class="service-item-heading">
+          Fast Shipping
+      </h2>
+      <p class="service-item-text">
+        We make sure you recieve your trombone as soon as we have finished making it. We also provide free returns if you are not satisfied.
+      </p>
+    </div>
+    </li>
+    <li class="services-item">
+       <div class="service-item-img">
+      <img src="https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcTti2je5jP4NJyie2Gjynf4ZklgkTOOGdloLQ&usqp=CAU" alt="">
+    </div>
+    <div class="service-item-desc">
+      <h2 class="service-item-heading">
+        Quality Assurance
+
+      </h2>
+      <p class="service-item-text">
+        For every purchase you make, we will ensure there are no damages or faults and we will check and test the pitch of your instrument.
+      </p>
+    </div>
+    </li>
+  </ul>
+</section>
+<section id="how-it-works" class="how-it-works">
+   <video src="https://media.istockphoto.com/id/621343474/video/orchestra-trumpet-on-the-background-of-the-demonstration.mp4?s=mp4-640x640-is&k=20&c=0wnRy-D_fQs8Ns5zua9JAxsgeokmF2SmL5UA-It0p60=" controls></video>
+</section>
+<section class="packages" id="packages">
+  <ul class="packages-list">
+    <li class="packages-item">
+      <h3 class="package-name">TENOR TROMBONE</h3>
+      <div class="package-price">
+        
+      </div>
+      <p>85$</p>
+      <p class="package-disc">Lorem ipsum. Lorem ipsum. Lorem ipsum doler. Lorem ipsum.</p>
+      <button class="package-button" >SELECT
+
+      </button>
+    </li>
+    <li class="packages-item">
+       <h3 class="package-name">BASS TROMBONE</h3>
+      <div class="package-price">
+        
+      </div>
+      <p>100$</p>
+      <p class="package-disc">Lorem ipsum. Lorem ipsum. Lorem ipsum dolor. Lorem ipsum.</p>
+      <button class="package-button" >SELECT
+        
+      </button>
+    </li>
+    <li class="packages-item">
+       <h3 class="package-name">VALVE TROMBONE</h3>
+      <div class="package-price">
+        
+      </div>
+      <p>120$</p>
+      <p class="package-disc">Plays similar to a Trumpet Great for Jazz Bands Lorem ipsum dolor. Lorem ipsum.</p>
+      <button class="package-button" >SELECT
+        
+      </button>
+    </li>
+  </ul>
+</section>
+<footer class="footer">
+  <ul class="footer-nav">
+    <li>Privacy</li>
+    <li>Terms</li>
+    <li>contact</li>
+  </ul>
```

Comment:
In footers you have to put the list items in <a> tag.

Reply:
- [ ]  Awesome observation! can you please rephrase your comment politely as a suggestion 💁‍♂️ and provide elaborate reason(s) for the required changes? It may be obvious to you and not so for others. 👍